bin/runner.js

```diff
@@ -215,19 +221,30 @@ function writeFile(testsDocPath) {
   };
 }
 
-function elmPathToModule(root, testsPath) {
+function elmPathToModule(elmRoot, sourceDirs, testsPath) {
   return function (pathName) {
-    var relativePath = path.relative(
-      path.resolve(path.join(testsPath, root)),
-      pathName
-    );
+    var relativePath = findModule(elmRoot, sourceDirs, pathName);
     if (relativePath.startsWith("./")) {
       relativePath = relativePath.substr(2);
     }
     return relativePath.substr(0, relativePath.length - 4).replace(/\//g, ".");
   };
 }
 
+function findModule(elmRoot, sourceDirs, pathName) {
+  for (var i = 0; i < sourceDirs.length; i++) {
+    const dir = sourceDirs[i];
+    const module = path.join(elmRoot, dir, pathName);
+    if (fs.existsSync(module)) {
+      return module;
+    }
```

Comment:
Optionally: Require exactly one match, failing explicitly if module reference is ambiguous (> 1 matches).

Reply:
I think elm make fails if there are more than one modules with the same name.